armbian/base/config/redis/redis.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Redis In-Memory Data Store
+After=network.target
+# original config in /lib/systemd/system/redis-server.service
+
+[Service]
+Type=forking
```

Comment:
Please test if this works fine when starting and stopping (restarting) Redis on Armbian. (I can't yet.)

I had a problem with `Type=forking`, Redis and systemd before. [Others aswell](https://stackoverflow.com/a/52335904). `Type=notify` could be the right way to go, when `forking` doesn't work.

Reply:
Current combo (supervised, daemonized + forking) runs fine, but I will try out the option mentioned in this link (supervised + notify) as well. Thanks.